sdk/storage/Azure.Storage.Blobs/tests/AppendBlobClientTests.cs

```diff
@@ -1004,6 +1004,43 @@ await foreach (BlobItem pathItem in test.Container.GetBlobsAsync())
             Assert.Contains(blobName, names);
         }
 
+        [Test]
+        public async Task OpenWriteAsync()
+        {
+            // Arrange
+            await using DisposingContainer test = await GetTestContainerAsync();
+
+            var blobName = GetNewBlobName();
+            AppendBlobClient blob = InstrumentClient(test.Container.GetAppendBlobClient(blobName));
+            await blob.CreateAsync();
+
+            AppendBlobOpenWriteOptions options = new AppendBlobOpenWriteOptions
+            {
+                BufferSize = Constants.KB
+            };
```

Comment:
We should test cases that cover more options. I.e. Overwrite. Here and in other blob types.

Reply:
I will, but I want to make sure we will actually ship this feature before I spend more time on unit tests.